README.md

```diff
@@ -134,7 +136,9 @@ A curated list of awesome mathematics resources.
 * [Paul's Online Math Notes](http://tutorial.math.lamar.edu/) - Paul Dawkins (Lamar University)
 * [List of electronic textbooks](http://faculty.atu.edu/mfinan/nnotes.html) - Marcel B. Finan (Arkansas Tech University)
 * [Topology Atlas](http://at.yorku.ca/topology/)
-
+* [Recreations in Math](http://djm.cc/library/Recreations_in_Mathematics_Licks_edited.pdf) - H. E. Licks
```

Comment:
Perhaps you could add the year for old scanned books like this one? e.g.

` * [Recreations in Math](http://djm.cc/library/Recreations_in_Mathematics_Licks_edited.pdf) - H. E. Licks (1917)`

Reply:
sure, happy to! can do so later today